Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -215,6 +217,12 @@ internal static string GetFullUrl(string host, int port)
             return builder.ToString();
         }
 
+        internal static bool ShouldUseV3CdpDebugger()
```

Comment:
Is this only called from the UI thread?

Reply:
Yep! This is called in: 

1) NodejsProjectLauncher.cs, which, when walking up the callstack, is ultimately fulfilling the launch functionality of . Assuming Microsoft.ViualSudio.Shell.Interop.IVsDebuggableProjectCfg, which is guaranteed UI thread.

2) JsFileDebugLaunchTargetProvider, which is called by Microsoft.VisualStudio.Workspace.Extensions.VS.Debug.IVsDebugLaunchTargetProvider, and assuming this IVsDebugLaunchTargetProvider is guaranteed UI thread, JsFileDebugLaunchTargetProvider usage is fine.

3) NodeJsDebugLaunchTargetProvider, and assuming Microsoft.VisualStudio.Workspace.Extensions.VS.Debug.IVsDebugLaunchTargetProvider2 is guaranteed UI thread, NodeJsDebugLaunchTargetProvider usage is fine